bucket/kyma-cli.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "1.21.0",
+    "description": "Simple set of commands to manage a Kyma installation",
+    "homepage": "https://github.com/kyma-project/cli",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kyma-project/cli/releases/download/1.21.0/kyma_Windows_x86_64.zip",
+            "hash": "46a261413ff74242599eca3b593694748e1b694bf58200e933759620ef1bfa0d"
+        },
+        "32bit": {
+            "url": "https://github.com/kyma-project/cli/releases/download/1.21.0/kyma_Windows_i386.zip",
+            "hash": "0ef097eff6e58890d66a539d77d37adfd81c7a81179dc3677f632f63dba2d2ac"
+        }
+    },
+    "bin": [
+        "kyma.exe"
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/kyma-project/cli/releases/download/$version/kyma_Windows_x86_64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/kyma-project/cli/releases/download/$version/kyma_Windows_i386.zip"
+            }
+        }
```

Comment:
Add hash extraction from checksums.txt

Reply:
Done in aa4176e. I have tested it locally using `kyma-cli@1.20.0` and it seems to work:

```
$ scoop install kyma-cli@1.20.0
WARN  Given version (1.20.0) does not match manifest (1.21.0)
WARN  Attempting to generate manifest for 'kyma-cli' (1.20.0)
Autoupdating kyma-cli
Searching hash for kyma_Windows_i386.zip in https://github.com/kyma-project/cli/releases/download/1.20.0/checksums.txt
Found: b2d7bae0f913f150b74891bbac29e631a03b330601b932c6a5684e5fb3519e33 using Extract Mode
Searching hash for kyma_Windows_x86_64.zip in https://github.com/kyma-project/cli/releases/download/1.20.0/checksums.txt
Found: 5f60dd184d7872ba5e2d78ab763dbc21641bfcfef470dd98dd1ce58114f3535e using Extract Mode
Writing updated kyma-cli manifest
Installing 'kyma-cli' (1.20.0) [64bit]
...
```

I am not familiar with the exact manifest syntax though so please check.